Release influxctl v2.6.0 (#5370)
influxdata/docs-v2: content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md

weight: 202
---

## v2.6.0 {date="2024-03-18"}

> `influxctl` 2.6.0 introduces the ability to create, list, and revoke
> management tokens. These tokens allow users to authenticate directly without the
> need to for OAuth2.

Reviewer: ```suggestion
`influxctl` 2.6.0 introduces the ability to create, list, and revoke
management tokens and allow you to authenticate directly with your
InfluxDB cluster instead of an OAuth2 provider.
```

### New Features

> - Add management tokens subcommands to create, list, and revoke management
>   tokens.
> - Introduce management token config option to authenticate using the management
>   tokens.

Reviewer: ```suggestion
- Add management token subcommands to create, list, and revoke management
  tokens.
- Introduce management token configuration option to authenticate using
  management tokens created with `influxctl`.
```
